eventhubs/client/README.md

```diff
@@ -0,0 +1,192 @@
+  have a separate view of the event stream, and to read the stream independently at their own pace and from their own
+  position. There can be at most 5 concurrent readers on a partition per consumer group; however it is recommended that 
+  there is only one active consumer for a given partition and consumer group pairing. Each active reader receives all of
+  the events from its partition; if there are multiple readers on the same partition, then they will receive duplicate 
+  events. 
+
+For more concepts and deeper discussion, see: [Event Hubs Features][event_hubs_features]. Also, the concepts for AMQP 
+are well documented in [OASIS Advanced Messaging Queuing Protocol (AMQP) Version 1.0][oasis_amqp_v1].
+
+## Examples
+will re-doc this section tonight
+- [Inspect Event Hub and partition properties][sample_get_event_hubs_metadata]
+- [Publish an event to an Event Hub][sample_send_event]
+- [Consume events from an Event Hub partition][sample_receive_event]
+
+## Troubleshooting
+
+### Enable client logging
+You can set the `AZURE_LOG_LEVEL` environment variable to view logging statements made in the client library. For example,
+setting `AZURE_LOG_LEVEL=2` would show all informational, warning, and error log messages. The log levels can be found here: [log levels][log_levels].
+
+### Enable AMQP transport logging
+If enabling client logging is not enough to diagnose your issues. You can enable logging to a file 
+in the underlying AMQP library, [Qpid Proton-J][qpid_proton_j_apache]. Qpid Proton-J uses `java.util.logging`. You can enable logging by create a configuration file with the contents below. Or set `proton.trace.level=ALL` and whichever configuration options you want for the `java.util.logging.Handler` implementation. Implementation classes and their options can be found in [Java 8 SDK javadoc][java_8_sdk_javadocs].
+
+#### Sample "logging.config" file ####
+``` logging config
+handlers=java.util.logging.FileHandler
+.level=OFF
+proton.trace.level=ALL
+java.util.logging.FileHandler.level=ALL
+java.util.logging.FileHandler.pattern=tracefilename
+java.util.logging.FileHandler.formatter=java.util.logging.SimpleFormatter
+java.util.logging.SimpleFormatter.format=[%1$tF %1$tr] %3$s %4$s: %5$s %n
+```
+
+### Common exceptions
+#### AMQP exception
+This is a general exception for AMQP related failures, which includes the AMQP errors as ErrorCondition and the context 
+that caused this exception as ErrorContext. 'isTransient' is a boolean indicating if the exception is a transient error
+or not. If true, then the request can be retried; otherwise not.
+
+- ErrorCondition: it contains constants common to the AMQP protocol and constants shared by Azure services. More detail
+  can be found in the link: [Event Hubs Messaging Exceptions][event_hubs_messaging_exceptions].
+- ErrorContext, it provides context that caused the AmqpException. The error occurs could be from AmqpConnection, 
```

Comment:
Remove this as a bullet item and use:

"The [`ErrorContext`](link to class) in the [AmqpException](link to class) provides information about the AMQP session, link, or connection that the exception occurred in. This is useful to diagnose which level in the transport this exception occurred at and whether it was an issue in one of the producers or consumers"

The content after: "The error occurs could be from AmqpConnection,  ...." can be removed. It's not helpful to people who haven't used the library. "What is SessionErrorContext?"